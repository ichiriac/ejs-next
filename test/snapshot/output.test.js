/**
 * Copyright (C) 2022 Ioan CHIRIAC (MIT)
 * @authors https://github.com/ichiriac/ejs-next/graphs/contributors
 * @url https://ejs.js.org
 */
"use strict";

const ejs = require("../../lib/ejs");

describe("Output", () => {
  it("Sanitize", (done, reject) => {
    ejs
      .render(`Hello <%= name %>`, { name: '<&">' })
      .then(function (output) {
        expect(output).toMatchSnapshot();
        done();
      })
      .catch(reject);
  });
});
