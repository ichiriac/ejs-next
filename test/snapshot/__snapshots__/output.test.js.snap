// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Output Sanitize 1`] = `"Hello &lt;&amp;&#34;&gt;"`;
